.github/workflows/ci.yml

```diff
@@ -17,7 +19,14 @@ jobs:
 
       - run: |
           if [ ! -z $(gofmt -l .) ]; then echo 'Make sure to run "go fmt ./..." before commit!' && exit 1; fi
-          go test ./...
+          mkdir  /home/runner/.supabase && chown -R runner /home/runner/.supabase
+          go test ./... -race -v -count=1 -cover -coverprofile=coverage.out
+          go tool cover -func coverage.out | grep total
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_PAT }}
```

Comment:
probably not needed

Reply:
This is needed for the goreleaser Action to push to supabase/homebrew-tap: https://github.com/goreleaser/goreleaser-action#limitation

Reply:
This is on ci.yaml, not release.yml.